addons/l10n_vn/data/account_tax_data.xml

```diff
@@ -2,83 +2,157 @@
+    <record id="tax_tag_sale_special_consumption" model="account.account.tag">
+        <field name="name">Special consumption tax</field>
+        <field name="applicability">taxes</field>
+    </record>
+
+    <!-- Fixed assets tags -->
+    <record id="tax_tag_purchase_fix_asset_0" model="account.account.tag">
+        <field name="name">Deductible Fixed Assets VAT 0%</field>
+        <field name="applicability">taxes</field>
+    </record>
+    <record id="tax_tag_purchase_fix_asset_5" model="account.account.tag">
+        <field name="name">Deductible Fixed Assets VAT 5%</field>
+        <field name="applicability">taxes</field>
+    </record>
+    <record id="tax_tag_purchase_fix_asset_10" model="account.account.tag">
+        <field name="name">Deductible Fixed Assets VAT 10%</field>
+        <field name="applicability">taxes</field>
+    </record>
+
+    <!-- Import tags -->
+    <record id="tax_tag_import_0" model="account.account.tag">
+        <field name="name">Import VAT 0%</field>
+        <field name="applicability">taxes</field>
+    </record>
+    <record id="tax_tag_import_5" model="account.account.tag">
+        <field name="name">Import VAT 5%</field>
+        <field name="applicability">taxes</field>
+    </record>
+    <record id="tax_tag_import_10" model="account.account.tag">
+        <field name="name">Import VAT 10%</field>
+        <field name="applicability">taxes</field>
+    </record>
+
+    <!-- Foreign import tags -->
+    <record id="tax_tag_foreign_import_0" model="account.account.tag">
+        <field name="name">Import tax 0%</field>
+        <field name="applicability">taxes</field>
+    </record>
+    <record id="tax_tag_foreign_import_5" model="account.account.tag">
+        <field name="name">Import tax 5%</field>
+        <field name="applicability">taxes</field>
+    </record>
+    <record id="tax_tag_foreign_import_10" model="account.account.tag">
+        <field name="name">Import tax 10%</field>
+        <field name="applicability">taxes</field>
+    </record>
+
+    <!-- Foreign export tags -->
+    <record id="tax_tag_foreign_export_0" model="account.account.tag">
+        <field name="name">Export tax 0%</field>
+        <field name="applicability">taxes</field>
+    </record>
+    <record id="tax_tag_foreign_export_5" model="account.account.tag">
+        <field name="name">Export tax 5%</field>
+        <field name="applicability">taxes</field>
+    </record>
+    <record id="tax_tag_foreign_export_10" model="account.account.tag">
+        <field name="name">Export tax 10%</field>
         <field name="applicability">taxes</field>
     </record>
 
     <!-- for purchase -->
     <record id="tax_purchase_vat10" model="account.tax.template">
         <field name="chart_template_id" ref="vn_template"/>
-        <field name="name">Thuế GTGT được khấu trừ 10%</field>
-        <field name="description">Thuế GTGT được khấu trừ 10%</field>
+        <field name="name">Deductible VAT 10%</field>
```

Comment:
Deductible is not the correct word. Not all Input VAT is deductible.

We should keep `Input VAT`.